MojioSDK/Models/Vehicles/Mojio.swift

```diff
@@ -16,6 +16,17 @@
 import Foundation
 import MojioCore
 
+public enum MojioType: String, Codable {
+    case obd2 = "Obd2"
+    case virtual = "Virtual"
+    case unknown = "Unknown"
+
+    public init(from decoder: Decoder) throws {
+        let label = try decoder.singleValueContainer().decode(String.self)
```

Comment:
Maybe `value` or `stringValue` is better?

Reply:
@andriimakukha  Thanks. It renamed to rawValue.